### ISSUE TITLE
Unable to find CVE data 

### ISSUE THREAD

**Issue by jo19in1** [plasticuproject/nvd_api#10]
1. I tried getting data for this CVE, getting an error:
`https://plasticuproject.pythonanywhere.com/nvd-api/v1/CVE-2021-3999
`
```
// 20220225110907
// https://plasticuproject.pythonanywhere.com/nvd-api/v1/CVE-2021-3999

{
  "message": "Internal Server Error"
}
```

**Comment by plasticuproject** [comment]
There is no match for `CVE-2021-3999` in NIST's NVD, this is why you are not receiving data back.
https://nvd.nist.gov/vuln/search/results?form_type=Basic&results_type=overview&query=CVE-2021-3999+&search_type=all&isCpeNameSearch=false

**Comment by jo19in1** [comment]
Ok got it. Where can I find the type of CVEs listed in my example. What would be a reliable source ro refer, like NVD?

**Comment by plasticuproject** [comment]
Actually, even though there was no matching CVE-ID, it should not have crashed the server like that. So this has been fixed with #12 

> _"Ok got it. Where can I find the type of CVEs listed in my example. What would be a reliable source ro refer, like NVD?"_

I'm not sure what you're asking here. If you have the CVE-ID, use that. If you don't have the CVE-ID, use the search functions to find the vulnerability you were looking for, using description keywords and/or year. I can't really think of a reason why you would need to search for CVEs just based on similar ID numbers.
